gix-object/src/commit/mod.rs

```diff
@@ -95,14 +99,14 @@ impl<'a> CommitRef<'a> {
     ///
     /// This is different from the `author` field which may contain whitespace.
     pub fn author(&self) -> gix_actor::SignatureRef<'a> {
```

Comment:
These must be fallible, panics aren't allowed. It's OK to use the error type returned by `SignatureRef::from_bytes()`.